kitty/debug_config.py

```diff
@@ -161,9 +241,9 @@ def debug_config(opts: KittyOpts) -> str:
     if is_macos:
         import subprocess
         p(' '.join(subprocess.check_output(['sw_vers']).decode('utf-8').splitlines()).strip())
-    if os.path.exists('/etc/issue'):
+    if is_linux and os.path.exists('/etc/issue'):
```

Comment:
/etc/issue might exist on other operating systems, so there should be a default print_issue() implementation.